### DISCUSSION TITLE
taking advantage of accessibility

### DISCUSSION THREAD

**Discussion by Hahado** [community/community#62187] [Other Feature Feedback, Questions, & Ideas] [answered]
### Select Topic Area

Question

### Body

there has been a person taking advantage of accessibility, by installing on all my apples devices which is giving them access to my device  i dont know how to stop..  sorry if done wrong

**Comment by iSemary** [comment]
Make sure you change your password to a strong password
Setup 2FA like Authenticator 
If it possible to change your email address it will be fine

wishing everything could be fine with you!

**Comment by queenofcorgis** [comment, marked as answer]
Hi @Hahado, Thanks for reaching out! 

We're sorry to hear you're experiencing this issue.

GitHub is a software hosting service, we host millions of projects, so probably every phone and computer on the planet has software on it that is also hosted here.

We're not able to provide support or answers for those individual projects. GitHub Support can only provide support for GitHub itself.

I'm sorry we can't help here but we'd recommend contacting a local IT specialist for assistance if you think your devices are affected by viruses or malware. They should be able to help you with these applications!
